packages/core-modules/contracts/modules/CoreElectionModule.sol

```diff
@@ -37,4 +45,58 @@ contract CoreElectionModule is IElectionModule, ElectionStorage, OwnableMixin {
     function getMemberTokenAddress() public view override returns (address) {
         return _electionStore().memberTokenAddress;
     }
+
+    function getNominees() external view override returns (address[] memory) {
+        return _electionStore().nominees;
+    }
+
+    function nominate() external override {
```

Comment:
@ajsantander is there any limit in the size of an address array that can be retrieved? 
line 49 will retrieve the whole array. Shall we limit the size here?

Reply:
@mjlescano confirmed. getNominees() will revert with out of gas if the address[] is too large.